plugins/modules/cohesity_job.py

```diff
@@ -1386,6 +1386,9 @@ def main():
                 job_details[
                     "parentSourceId"
                 ] = get__prot_source_root_id__by_environment(module, job_details)
+                if job_details["environment"] == "VMware":
```

Comment:
In the bug we dont seem to get the parent id, can we gracefully fail if this is not caught